docs/source/install.ipynb

```diff
@@ -16,6 +16,8 @@
     "pip install evalml\n",
     "```\n",
     "\n",
+    "For Python versions 3.6.X you must install `pip >= 18.0` and `setuptools > 40.0.0` in order to install evalml.\n",
```

Comment:
What happens when a user runs `pip install evalml` but has pip below this version, or setuptools below this version? Should we update our requirements with this?

Reply:
(We don't currently list pip or setuptools as dependencies)

Reply:
If the user is using 3.6, they run into the `shap` error that caused this issue. I figured adding it into the install docs would be easiest since `pip` and `setuptools` aren't in our requirements docs

Reply:
Not sure if this is 100% relevant, but I'd like point out that creating a virtual env with 3.6 directly means we don't run into this issue:
```
python3.6 -m pip install virtualenv
python3.6 -m virtualenv my-venv --python=python3.6
source my-venv/bin/activate
python3.6 -m pip install shap (or -r requirements.txt in my case)
```

This works fine. Could be because of the versions of pip and setuptools that are installed with the virtualenv but didn't dig too deeply.

Reply:
Hmm yeah that's interesting. Looking further into it, when I used @angela97lin's approach, `pip == 20.2.3` and `setuptools == 50.3.1`, as opposed to using pyenv to create a virtualenv for Python3. I think both approaches are valid since the versions of `pip` and `setuptools` are high enough. I think with the current message that I added, it should be enough to allow users to install `evalml` without issues. @dsherry Do you think these two should be added to our dependencies, or would leaving/updating the message be enough?

Reply:
@bchen1116 so as I just wrote on the issue, when I create a pyenv with `3.6.10` and run `pip install .`, it passes. But that fails with `3.6.0`.

Can we update this to say "For older Python versions (3.6.0 in particular), you must install `pip>=18.0` and `setuptools>=40.0.0` before installing evalml."

Great find!